### PR TITLE
feat(docs & passwords and secrets): consideration of kics-scan ignore command and LinesIgnore 

### LINKED ISSUE #4419: Kics reports an error on Ansible commented values
### Expected Behavior

**Kics** should not report that a he finds a password in my file.

### Actual Behavior

Beside my value is just an example and is commented, **kics** reports an error

### Steps to Reproduce the Problem

1.  Have the following **group_vars/backends/vars.yml**

```yaml
---
backends:
  db_name: "db"
  user:
    username: "user"
#    definition of using host_vars variable in an other variable
#    hostvars:
#      airflow_1:
#        vault_demo_airflow_user_1_password: value
    password: "{{ another_encrypted_value }}"

```

2.  In `checkmarx/kics:debian` run the analysis `/app/bin/kics scan -p group_vars/backends/vars.yml -o ${PWD} --report-formats json,sarif --ignore-on-exit results --verbose --log-level=DEBUG`

3.  [Obtain this kind of output ](https://github.com/Checkmarx/kics/files/7394785/analysis.txt)

### Specifications

-   Version: 1.4.5
-   Platform: Ansible
-   Subsystem: N/A

### LINKED ISSUE #4485: KICS should not load all queries when it does not recognize a platform
### Expected Behavior
KICS does not load all queries when it does not recognize a platform

### Actual Behavior
KICS loads all queries when it does not recognize a platform

### Steps to Reproduce the Problem

1.  Run KICS against any unsupported file. For example, a go file.
2.  Observe the value of  `Queries loaded:`

-   Version: Development
-   Subsystem: Windows

### PR DESCRIPTION
**Pull request by rafaela-soares**
Closes #4485
Closes #4419

**Proposed Changes**
- Added documentation about Ansible identification limitation
- Added consideration of `kics-scan ignore` command in passwords and secrets inspector
- Added consideration of LinesIgnore in passwords and secrets inspector (Fix bug #4419)
- Fix bug #4485

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: pkg/engine/secrets/inspector.go [hunk 1]
```diff
@@ -168,22 +168,24 @@ func (c *Inspector) Inspect(ctx context.Context, basePaths []string,
...
+						continue
 					}
-					continue
-				}
 
-				// check file content as a whole
-				c.checkFileContent(&c.regexQueries[i], basePaths, &files[idx])
+					// check file content as a whole
+					c.checkFileContent(&c.regexQueries[i], basePaths, &files[idx])
+				}
```
Comment (joaoReigota1) [inline pkg/engine/secrets/inspector.go:188]: Maybe we should place this piece of code in a separate function for readability what do you think?

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]
Please Check comments

File: pkg/engine/secrets/inspector.go (hunk 1 above)
Comment (rafaela-soares) [reply pkg/engine/secrets/inspector.go:188]: I agree 🚀

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Pull request merged**